android/src/main/java/com/tailscale/ipn/MainActivity.kt

```diff
@@ -267,7 +267,9 @@ class MainActivity : ComponentActivity() {
   override fun onNewIntent(intent: Intent?) {
     super.onNewIntent(intent)
     if (intent?.getBooleanExtra(START_AT_ROOT, false) == true) {
-      navController.popBackStack(route = "main", inclusive = false)
+      if (this::navController.isInitialized) {
```

Comment:
Drive-by, @agottardo noticed a crash here when navController is not initialized. If it's not initialized, that means we're just starting the activity, in which case we'll end up on `main` anyway.

Reply:
Moved to its own PR - https://github.com/tailscale/tailscale-android/pull/386